README.rst

```diff
@@ -101,43 +119,50 @@ Now go to the ``data-model`` directory and run the ``./init_qwat.sh`` script:
 
 The script has the following options:
 
-- ``-p``                   PG service to connect to the database.
+- ``-p``                     PG service to connect to the database.
 - ``-s`` or ``--srid``         PostGIS SRID. Default to 21781 (ch1903)
-- ``-d`` or ``--drop-schema``  drop schemas (cascaded) if they exist
+- ``-d`` or ``--drop-schema``    drop schemas (cascaded) if they exist
 - ``-r`` or ``--create-roles`` create roles in the database
 
-You can restore a sample dataset, then download the data sample and insert it into the qwat database:
+Optionally, You can restore a sample dataset, then download the data sample and insert it into the QWAT database:
```

Comment:
s/You/you

Reply:
s/then/. For that/

Reply:
I'd say: "For that you need to download the data sample dump and restore it into the QWAT database:".

Reply:
Thanks.